string location_name
string parent_location
---
string status_message
